src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form class="search-form search-form_big" data-qa="big">
+      <label class="search-form__label search-form__label_big">
+        <input data-qa="keypress"
+        class="search-form__input search-form__input_big"
```

Comment:
```suggestion
    <form class="search-form" data-qa="big">
      <label class="search-form__label">
        <input data-qa="keypress"
        class="search-form__input"
```
big modifier classes are redundant.
set all properties for block or element classes (search-form, search-form__label, search-form__input) and reset necessary for small modifiers